thunder/tests/opinfos.py

```diff
@@ -6152,6 +6152,72 @@ def tensor_1d_sample_generator(op, device, dtype, requires_grad, **kwargs):
 nn_ops = []
 
 
+def baddbmm_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    cases = (
+        ((0, 0, 0), (0, 0, 0), (0, 0, 0)),
+        ((3, 0, 5), (3, 0, 0), (3, 0, 5)),
+        ((0, 5, 6), (0, 5, 0), (0, 0, 6)),
+        ((3, 5, 6), (3, 5, 0), (3, 0, 6)),
+        ((3, 5, 6), (3, 5, 8), (3, 8, 6)),
+    )
+
+    constants_cases = (
+        (2.0, 2.0),
+        (1.0, 0.0),
+        (0.0, 1.0),
+    )
```

Comment:
`float('inf')`, `float('-inf')`, `float('nan')` are also important to check besides 0.0, 1.0, and 2.0. Could you please add them?